book/animations.md

```diff
@@ -2313,10 +2327,8 @@ raster either.
 
 [tr-example]: examples/example13-transform-transition.html
 
-13-4 *Width animations*. Implement the CSS `width` and `height` properties; when
-`width` is set to some number of pixels on an element, the element should be
-that many pixels wide, regardless of how its width would normally be computed;
-the same goes for `height`. Make them animatable; you'll need a variant of
+13-4 *Width/height animations*. (You'll need to have done Exercise 6-2 first.)
+Make `width` and `height` animatable; you'll need a variant of
```

Comment:
In previous chapters the "do Exercise X-Y first" text comes after the first sentence before the hints, and it's not parenthesized. Can we do that here?